config/serverless.yml

```
@@ -28,6 +28,10 @@ xpack.remote_clusters.enabled: false
 xpack.snapshot_restore.enabled: false
 xpack.license_management.enabled: false
 
+# Keep deeplinks visible so that they are shown in the sidenav
+dev_tools.deeplinks.visible: true
```

nit: since the type is `status`, you might consider matching this to the API.

```suggestion
dev_tools.deeplinks.navLinkStatus: visible
management.deeplinks.navLinkStatus: visible
```
```